### DISCUSSION TITLE
How can we import stickers ?

### DISCUSSION THREAD

**Discussion by ghost** [phase1geo/Minder#574] [Q&A]
Hello, everybody and thanks for this great software.

Just, I can’t find how to import my own stickers and them category.

I don’t find where stickers are stocked anywhere… (Debian)

That’s all, folks !

**Comment by phase1geo** [comment]
Currently, stickers cannot be imported into Minder.  Stickers are stored as a resource within the application (they aren't stored in the filesystem) which is why you probably can't find them there.

With that said, I do plan on adding the ability to bring user-provided stickers into the application (and being able to categorize them).  This capability will probably be in the application whenever the 2.0 version drops.  Though there is no date planned for when 2.0 will be available.  This feature request is already being tracked in #556.
